[BUG] Error installing from git

Hello, when I install Fabuya using: `yarn add git+https://github.com/HanzHaxors/fabuya.git` and using it.
Error: `Cannot find module './baileys-lib'` in `'node_modules/fabuya/index.js'`
SAlu2s

--- a/build.sh
+++ b/build.sh
@@ -6,8 +6,6 @@ npm run build:baileys
 
 # Copy Baileys runtime dependencies
 cp -rf ./Baileys/WAProto ./WAProto
-cp -rf ./Baileys/WABinary ./WABinary
-cp -rf ./WABinary/Constants.js ./baileys-lib/Constants.js
 cp -rf ./Baileys/WASignalGroup ./WASignalGroup
 
 # TODO: update Baileys and see the effect

--- a/package.json
+++ b/package.json
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "build:baileys": "tsc -p ./Baileys --outDir ./baileys-lib",
-    "init:baileys": "npm i -D --prefix ./Baileys"
+    "init:baileys": "npm i -D --prefix ./Baileys",
+    "postinstall": "node postinstall.js"
   },
   "repository": {
     "type": "git",

--- a/postinstall.js
+++ b/postinstall.js
@@ -1,0 +1,13 @@
+const fs = require('fs');
+const { spawnSync } = require('child_process');
+
+// Initialize git submodule
+spawnSync('git', ['submodule',  'update', '--init', '-N', 'Baileys']);
+spawnSync('npm', ['install', '-D']);
+spawnSync('npm', ['install', '-D', '--prefix ./Baileys']);
+
+// Move directories
+fs.renameSync('./Baileys/lib', './baileys-lib');
+
+fs.renameSync('./Baileys/WAProto', './WAProto');
+fs.renameSync('./Baileys/WASignalGroup', './WASignalGroup');

